portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java

```diff
@@ -405,6 +384,11 @@ public void validateFriendlyURL(
 			return;
 		}
 
+		if (_layoutFriendlyURLEntryValidator != null) {
```

Comment:
I moved this section of the code back to the getFriendlyURL() method, instead of inserting it in validateFriendlyURL(), and that resolved the issue.
Should I make that adjustment and send it to the team to review?